$OpenBSD$

fix arpa/inet.h:74: warning: 'struct in_addr' declared inside parameter list

--- src/lib/ecore_con/ecore_con_ares.c.orig	Sat Oct  8 18:49:44 2011
+++ src/lib/ecore_con/ecore_con_ares.c	Wed Oct 26 03:49:03 2011
@@ -10,6 +10,11 @@
  */
 
 #include <string.h>
+#include <sys/types.h>
+
+#ifdef HAVE_NETINET_IN_H
+# include <netinet/in.h>
+#endif
 
 #ifdef HAVE_ARPA_INET_H
 # include <arpa/inet.h>
